dash-pipeline/bmv2/dash_acl.p4

```diff
@@ -29,9 +29,10 @@ match_kind {
     table table_name { \
         key = { \
             meta. ## table_name ##_dash_acl_group_id : exact @name("meta.dash_acl_group_id:dash_acl_group_id"); \
-            hdr.ipv4.dst_addr : LIST_MATCH @name("hdr.ipv4.dst_addr:dip"); \
-            hdr.ipv4.src_addr : LIST_MATCH @name("hdr.ipv4.src_addr:sip"); \
-            hdr.ipv4.protocol : LIST_MATCH @name("hdr.ipv4.src_addr:protocol"); \
+            meta.is_overlay_ip_v6 : exact @name("meta.is_overlay_ip_v6:is_overlay_pkt_ip_v6"); \
```

Comment:
I don't think we need the is_overlay_pkt_ip_v6 field here in each ACL rule since unlike routing we have separate ACL groups for v4 and v6. So depending on whether the pkt is v4 or v6, the correct ACL group is picked up by the pipeline. And all entries in that group will have only v4 or only v6 but not both.

Reply:
Hi @mukeshmv, thanks for reviewing it.  Are you mixing inbound/outbound ACL groups with v4/v6?  Where (or how) in the pipeline it picks up the correct ACL group based on the v4/v6 packet?  May be I am missing something here?  Thanks!

Reply:
@mhanif pipeline picks up v4 or v6 ACL group here
https://github.com/Azure/DASH/blob/main/dash-pipeline/bmv2/dash_pipeline.p4#L120

Reply:
Hi @mukeshmv, Yes, you are right.  I have made the requested changes and resubmitted to this PR.  Thanks.